test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CustomHandler.cs

```diff
@@ -52,7 +52,7 @@ private async Task InvokeHttpTrigger(string functionName)
 
             string responseContent = await response.Content.ReadAsStringAsync();
             JObject res = JObject.Parse(responseContent);
-            Assert.True(res["functionName"].ToString().StartsWith(functionName));
+            Assert.True(res["functionName"].ToString().StartsWith($"api/{functionName}"));
```

Comment:
>$"api/{functionName}" [](start = 66, length = 21)

Wanted to double check this. Since it's custom handler and EnableRequestForwarding was set to true in this scenario, so call was getting routed to "api/functionName".

Reply:
Correct! Thanks!